src/client/pfs/pfs.proto

```diff
@@ -613,6 +613,7 @@ service API {
   rpc GetTar(GetTarRequest) returns (stream google.protobuf.BytesValue) {}
   // Refer to the GetTarConditionalRequest / GetTarConditionalResponse message definitions for the protocol.
   rpc GetTarConditional(stream GetTarConditionalRequest) returns (stream GetTarConditionalResponse) {}
+  rpc ListFileNS(ListFileRequest) returns (stream FileInfoNewStorage) {}
```

Comment:
I think it would be better to name this `ListFileNewStorage` because I think it will be unclear to others what `NS` means.

Reply:
What do you think about standardizing around the suffix "2" or something less verbose? If this is expected by default in 2.0, then that seems like a reasonable naming convention.

Reply:
Yeah, it is a bit verbose. I think moving over to a suffix such as `V2` would work. I had some concerns about the fact that we are probably going to have three parallel configurations (1.x, 1.x with new storage, and 2.x), but the new endpoint interfaces should be the same in 2.x. Two feature flags should be enough to navigate the 1.x with new storage and 2.x branches.